test/e2e-common.sh

```diff
@@ -127,3 +175,58 @@ function dump_extra_cluster_state() {
     done
   done
 }
+
+function install_istio {
+  if [[ -z "${ISTIO_VERSION}" ]]; then
+    readonly ISTIO_VERSION="1.4.3"
+  fi
+  echo ">> Installing Istio: ${ISTIO_VERSION}"
+
+  local istio_base="./third_party/istio-${ISTIO_VERSION}"
+  INSTALL_ISTIO_CRD_YAML="${istio_base}/istio-crds.yaml"
+  INSTALL_ISTIO_YAML="${istio_base}/istio-minimal.yaml"
+
+  echo "Istio CRD YAML: ${INSTALL_ISTIO_CRD_YAML}"
+  echo "Istio YAML: ${INSTALL_ISTIO_YAML}"
+
+  echo ">> Bringing up Istio"
```

Comment:
Testing eventing should not need istio

Reply:
Istio here is for serving. Let's discuss that in https://github.com/knative/eventing/pull/2388#discussion_r385882618